README.md

```diff
@@ -172,41 +172,43 @@ Running the EVM gateway for mainnet requires additional security and stability m
+| `coa-key`                      | `""`                          | Private key for the COA address used for transactions                                    |
+| `coa-key-file`                 | `""`                          | Path to a JSON file of COA keys for key-rotation (exclusive with `coa-key` flag)         |
+| `coa-resource-create`          | `false`                       | Auto-create the COA resource if it doesn't exist in the Flow COA account                 |
+| `coa-cloud-kms-project-id`     | `""`                          | Project ID for KMS keys (e.g. `flow-evm-gateway`)                                        |
+| `coa-cloud-kms-location-id`    | `""`                          | Location ID for KMS key ring (e.g. 'global')                                             |
+| `coa-cloud-kms-key-ring-id`    | `""`                          | Key ring ID for KMS keys (e.g. 'tx-signing')                                             |
+| `coa-cloud-kms-keys`           | `""`                          | KMS keys and versions, comma-separated (e.g. `"gw-key-6@1,gw-key-7@1"`)                  |
+| `log-level`                    | `debug`                       | Log verbosity level (`debug`, `info`, `warn`, `error`, `fatal`, `panic`)                 |
+| `log-writer`                   | `stderr`                      | Output method for logs (`stderr`, `console`)                                             |
+| `stream-limit`                 | `10`                          | Rate-limit for client events sent per second                                             |
+| `rate-limit`                   | `50`                          | Requests per second limit for clients over any protocol (ws/http)                        |
+| `address-header`               | `""`                          | Header for client IP when server is behind a proxy                                       |
+| `heartbeat-interval`           | `100`                         | Interval for AN event subscription heartbeats                                            |
+| `stream-timeout`               | `3`                           | Timeout in seconds for sending events to clients                                         |
+| `force-start-height`           | `0`                           | Force-set starting Cadence height (local/testing use only)                               |
+| `wallet-api-key`               | `""`                          | ECDSA private key for wallet APIs (local/testing use only)                               |
+| `filter-expiry`                | `5m`                          | Expiry time for idle filters                                                             |
+| `traces-gcp-bucket`            | `""`                          | GCP bucket name for transaction traces                                                   |
```

Comment:
I just noticed something that could be problematic, though not really related to this PR.
The `traces-gcp-bucket` accepts a single bucket name, but for `mainnet` we already have 2:
- `mainnet26-evm-execution-traces1`
- `mainnet25-evm-execution-traces1`

We need to make sure that when we back-fill, the provided heights exist in the appropriate bucket, but I don't know what is contained in those 2 buckets.

Reply:
I added a check that will log a warning if the start height is before the init-cadence height ([378e60f](https://github.com/onflow/flow-evm-gateway/pull/615/commits/378e60f86d834fba4bced1f3496cd310cbcef52b)). this may not work for nodes that have been up for a while, but not sure what else we can do without adding a bunch of complexity to the startup logic